tests/test_helper.js

```diff
@@ -1,298 +1,99 @@
       ],
     },
-  ],
-  ingredients: [
     {
-      ingredient: "eggs",
-      unit: "none",
-      amount: 3,
-      section: "cake",
+      stepName: "Whipped Cream",
+      step: [
+        "In a bowl, whisk the cream and ¼ cup (60 ml) of the condensed milk with an electric mixer until stiff peaks form. Transfer ½ cup (125 ml) of the whipped cream into a pastry bag fitted with a plain tip. Set aside until ready to decorate the cake.",
+      ],
     },
     {
-      ingredient: "sugar",
-      unit: "cup",
-      amount: 1 / 4,
-      section: "cake",
+      stepName: "Assembly",
+      step: [
+        "Place one cake on a serving dish. Using a pastry brush, cover the top of the cake with the remaining condensed milk. Spread ¾ cup (180 ml) of the whipped cream over the top of the cake. Top with the diced strawberries. Cover with the second cake. Using a spatula, spread ¾ cup (180 ml) of the whipped cream in a thin layer around the sides of the cake. Spread the remaining whipped cream over the top of the cake. Using the pastry bag, pipe dollops of the reserved whipped cream around the top edge of the cake. Decorate with the fresh strawberries. The strawberry shortcake will keep covered in the refrigerator for 2 days.",
+      ],
     },
+  ],
+  ingredients: [
     {
-      ingredient: "vegetable oil",
-      unit: "tbsp",
-      amount: 2,
       section: "cake",
+      items: [
+        {
+          name: "eggs",
+          unit: "none",
+          amount: 3,
+        },
+        {
+          name: "milk",
+          unit: "cup",
+          amount: 1,
+        },
+        {
+          name: "sugar",
+          unit: "cup",
+          amount: 0.5,
+        },
+      ],
+    },
+    {
+      section: "Whipping Cream",
+      items: [
+        {
+          name: "33% Whipping Cream",
+          unit: "cup",
+          amount: 2,
+        },
+        {
+          name: "Icing sugar",
+          unit: "cup",
+          amount: 0.5,
+        },
+        {
+          name: "Condensed Milk",
+          unit: "cup",
+          amount: 0.5,
+        },
+      ],
     },
   ],
   time: {hours: "1", minutes: "30"},
   tags: ["cake", "strawberry"],
 };
 
+const initialRecipes = [testRecipe, testRecipe, testRecipe];
+
+const recipeToAdd = {
+  ...testRecipe,
+  title: "strawberry shortcake ver 5",
+  picturePath:
+    "https://www.google.com/imgres?imgurl=https%3A%2F%2Fimages.ricardocuisine.com%2Fservices%2Frecipes%2F9640.jpg&tbnid=eFe18pfPtTruUM&vet=12ahUKEwjbw9ybiO2BAxWSBzQIHd8WAyoQMygOegUIARCWAQ..i&imgrefurl=https%3A%2F%2Fwww.ricardocuisine.com%2Fen%2Frecipes%2F9640-japanese-strawberry-shortcake&docid=K_d7KXa2YhtFcM&w=1920&h=2592&q=strawberry%20shortcake%20japanese&ved=2ahUKEwjbw9ybiO2BAxWSBzQIHd8WAyoQMygOegUIARCWAQ",
```

Comment:
You can use a dummy url for a test. URLs like this are not good. Very hard to read.